Kickstarter-iOS/Views/Controllers/ProjectNavigatorViewController.swift

```diff
@@ -188,6 +188,10 @@ extension ProjectNavigatorViewController: ProjectPamphletViewControllerDelegate
 
     let navigationController = UINavigationController(rootViewController: rewardsViewController)
 
+    if AppEnvironment.current.device.userInterfaceIdiom == .pad {
+      navigationController.modalPresentationStyle = .formSheet
```

Comment:
Looks like the rest of our codebase doesn't really bother checking for this...I wonder if we should just omit this check and always set it to `formSheet`...it will only apply on iPads based on the docs:
```
The presentation style determines how a modally presented view controller is displayed on the screen. On iPhone and iPod touch devices, modal view controllers are always presented full-screen, but on iPad devices there are several different presentation options. For a list of possible presentation styles, and their compatibility with the available transition styles, see the “UIModalPresentationStyle” constant descriptions.
```

Reply:
Sure 👍